src/www/containers/contactpage/ContactPage.scss

```diff
@@ -0,0 +1,164 @@
+    padding-bottom: 4rem;
+    width: 100%;
+
+    form{
+        display: flex;
+        flex-direction: column;
+        margin-top: 2vw;
+        align-items: center;
+
+        Input, textarea{
+            border: 1px solid $softBlack;
+            border-radius: 20px;
+            box-shadow: 1px 1px $softBlack;
+            font-family: $globalFonts;
+            font-size: 1.2rem;
+            height: 2rem;
+            margin-top: 1vh;
+            margin-left: 5rem;
+            margin-right: 5rem;
+            margin-bottom: 2vh;
+            outline: none;
+            padding: 0.75rem 1rem;
+            padding-left: 1.75rem;
+            width: 20rem;
+            word-break: break-all;
+        }
+
+        textarea{
+            height: 20vh;
+            resize: none;
+        }
+
+        .submit-btn{
+            cursor: pointer;
+            font-family: $globalFonts;
+            font-size: 1.5rem;
+            transition: 0.1s ease-in-out;
+            width: 10rem;
+            height: 4rem;
+            border: 1px solid rgba(0, 0, 0, 0.1);
+            border-radius: 0.5rem;
+            box-shadow: 1px 1px rgba(0, 0, 0, 0.1);
+            margin-bottom: 1rem;
+            outline: none;
+            padding: 0.75rem;
+    
+          &:hover {
+            background-color: $softBlack;
+            }
+        }
+    }
+
+    .contact-form-block{
+        background-color: white;
+        border: 1px solid $softBlack;
+        border-radius: 2vh;
+        display: flex;
+        flex-direction: row;
+        height: 35rem;
+    }
+
+    .contact-form-title1{
+        color: black;
+        font-family: $globalFonts;
+        font-size: 3.1rem;
+        margin-top: 3.8vh;
+        line-height: 1.2;
+        text-align: center;
+        padding-bottom: 7px;
+    }
+
+    .contact-form-title2{
+        color: black;
+        font-family: $globalFonts;
+        font-size: 2rem;
+        line-height: 1.5;
+        text-align: center;
+        padding-bottom: 1vh;
+    }
+
+    .form-image{
+        border-radius: 2vh;
+    }
+
+    .title-block {
+        align-items: center;
+        display: flex;
+        margin-top: 2vh;
+        flex-direction: column;
+        top: 2vw;
+        color: black;
+        width: 100%;
+    }
+
+    .top-img{
+        position: absolute;
+        width: 100%;
+        z-index: -1;
+    }
+}
+
+@media only screen and(max-width: 1300px) {
+    .form-image{
+        display: none;
+    }
+}
+
+@media only screen and(max-width: 900px) {
+
+    .contact-form-block{
+        margin-top: 1vh;
+    }
+
+    .contact-form-title2{
+        display: none;
+    }
+
+  }
+
+@media only screen and(max-width: 600px) {
+    .container{
+
+        form{
+            width: 25rem;
+        }
+
+        .contact-form-block{
+            border: none;
+        }
+
+        .contact-form-title1{
+            padding-bottom: 0;
+        }
+
+        .top-img{
+            width: 100%;
+            height: 15rem;
```

Comment:
Fixed height is causing the top image to deform when the width is <= 600px.
Should we follow a different approach and just have a full-height background image?